consensus/docker-compose-files/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN mkdir -p /var/hyperledger/db
 RUN mkdir -p /var/hyperledger/production
 WORKDIR $GOPATH/src/github.com/hyperledger/fabric/
 COPY . .
+WORKDIR $GOPATH/src/github.com/hyperledger/fabric/peer
+RUN pwd
```

Comment:
Hi @christo4ferris, thanks for pointing it out. The 'pwd' command is actually not needed. I will remove it.